docs/resources/kafka.md

```diff
@@ -313,6 +301,18 @@ Read-Only:
 - `ssl` (Boolean)
 - `usage` (String)
 
+
+<a id="nestedatt--kafka"></a>
+### Nested Schema for `kafka`
+
+Read-Only:
+
+- `access_cert` (String)
```

Comment:
it no longer says `Sensitive`

Reply:
But I can't make it use that word. Or can I?

Reply:
why doesn't it works anymore 🤔